delivery.yaml

```diff
@@ -0,0 +1,42 @@
+build_steps:
+    - desc: Prepare Environment
+      cmd: |
+        apt-get update
+        apt-get install -y python3 python3-pip jq
```

Comment:
Are Python 3 and pip necessary in the builder VM for this build?

Reply:
They are needed only for scm-source